airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -335,12 +336,96 @@ async def get_step_result(self, container: Container) -> StepResult:
             status=self.get_step_status_from_exit_code(exit_code),
             stderr=stderr,
             stdout=stdout,
-            output=container,
+            output={"report_log": report_log_artifact},
             artifacts=[report_log_artifact],
             consider_in_overall_status=consider_result_in_overall_status,
         )
 
 
+class IncrementalAcceptanceTests(Step):
```

Comment:
The naming off this is great! I like that it conveys that "The acceptance tests failed" (`Ignored - Failed`) and "the incremental acceptance tests succeeded", and the latter is what we're making decisions based on.